src/renderers/dom/client/__tests__/ReactEventListener-test.js

```diff
@@ -36,6 +36,26 @@ describe('ReactEventListener', function() {
     ReactEventListener._handleTopLevel = handleTopLevel;
   });
 
+  it('should dispatch events from outside React tree', function() {
+    var otherNode = document.createElement('h1');
+    var component = ReactMount.render(<div />, document.createElement('div'));
+    expect(handleTopLevel.mock.calls.length).toBe(0);
+    ReactEventListener.dispatchEvent(
+      'topMouseOut',
+      {
+        type: 'mouseout',
+        fromElement: otherNode,
+        target: otherNode,
+        srcElement: otherNode,
+        toElement: React.findDOMNode(component),
+        relatedTarget: React.findDOMNode(component),
+        view: window,
+        path: [otherNode, otherNode],
```

Comment:
Yeah, it's not, but it's a mock event anyway and close enough to be sufficient for the unit test.